mp/src/game/shared/shareddefs.h

```diff
@@ -85,17 +85,25 @@ class CViewVectors
 #define VEC_DEAD_VIEWHEIGHT	g_pGameRules->GetViewVectors()->m_vDeadViewHeight
 
 #ifdef NEO
-#define NEO_RECON_VIEW_OFFSET Vector(0.0f, 0.0f, -6.0f)
-#define NEO_ASSAULT_VIEW_OFFSET Vector(0.0f, 0.0f, -3.75f)
-#define NEO_SUPPORT_VIEW_OFFSET Vector(0.0f, 0.0f, 3.0f)
+#define NEO_RECON_VIEW_OFFSET Vector(0.0f, 0.0f, -1.f)
+#define NEO_ASSAULT_VIEW_OFFSET Vector(0.0f, 0.0f, -0.5f)
+#define NEO_SUPPORT_VIEW_OFFSET Vector(0.0f, 0.0f, 2.0f)
+#define NEO_RECON_DUCKED_VIEW_OFFSET Vector(0.0f, 0.0f, -4.f)
+#define NEO_ASSAULT_DUCKED_VIEW_OFFSET Vector(0.0f, 0.0f, -4.f)
+#define NEO_SUPPORT_DUCKED_VIEW_OFFSET Vector(0.0f, 0.0f, 5.f)
+
 #define VEC_VIEW_NEOSCALE(NeoPlayer) (g_pGameRules->GetViewVectors()->m_vView + ((NeoPlayer->GetClass() == NEO_CLASS_RECON) ? NEO_RECON_VIEW_OFFSET : (NeoPlayer->GetClass() == NEO_CLASS_SUPPORT) ? NEO_SUPPORT_VIEW_OFFSET : NEO_ASSAULT_VIEW_OFFSET))
-#define VEC_DUCK_VIEW_NEOSCALE(NeoPlayer) (g_pGameRules->GetViewVectors()->m_vDuckView + ((NeoPlayer->GetClass() == NEO_CLASS_RECON) ? NEO_RECON_VIEW_OFFSET : (NeoPlayer->GetClass() == NEO_CLASS_SUPPORT) ? NEO_SUPPORT_VIEW_OFFSET : NEO_ASSAULT_VIEW_OFFSET))
+#define VEC_DUCK_VIEW_NEOSCALE(NeoPlayer) (g_pGameRules->GetViewVectors()->m_vDuckView + ((NeoPlayer->GetClass() == NEO_CLASS_RECON) ? NEO_RECON_DUCKED_VIEW_OFFSET : (NeoPlayer->GetClass() == NEO_CLASS_SUPPORT) ? NEO_SUPPORT_DUCKED_VIEW_OFFSET : NEO_ASSAULT_DUCKED_VIEW_OFFSET))
```

Comment:
Nitpick: These which maps class to a value could be refactored to utilize a const array to map class to offset instead.
Although, not ground for blocking PR/can be ignored.

Reply:
![image](https://github.com/user-attachments/assets/38ce8c40-1cbe-4951-b286-e1e53fd57f4d)
Could probably be refactored further to use this NEOViewVectors object which is closer to how the base game works. If it's alright we can do that in a separate pr?

Reply:
It's fine to do it as a separate PR